metroae

```diff
@@ -677,42 +677,85 @@ function destroy {
 
 function pull {
     debug ${FUNCNAME[0]}
-    if [[ ! -z $1 ]]; then
-        MAX_CONTAINER_VERSION=$1
-    fi
-    debug "${FUNCNAME[0]}: MAX_CONTAINER_VERSION: $MAX_CONTAINER_VERSION"
+    if [[ ! -z $1 ]] && [[ $1 == "--repo" ]]; then
+        REPO=$2
+        debug "${FUNCNAME[0]}: REPO: $REPO"
+        
+        set -e
 
-    set -e
+        echo ""
+        write_to_screen_and_script_log ">>> Pulling the MetroAE container image from Docker hub"
```

Comment:
We're not pulling from dockerhub in this case right? Maybe we should log the repo name